requirements/edx/base.txt

```diff
@@ -87,7 +87,7 @@ cachecontrol==0.14.0
     # via firebase-admin
 cachetools==5.5.0
     # via google-auth
-camel-converter[pydantic]==3.1.2
+camel-converter[pydantic]==4.0.1
```

Comment:
Version `4.0.0` dropped support for `Python 3.8`. [[CHANGELOG](https://github.com/sanders41/camel-converter/releases/tag/v4.0.0)]